src/containers/auth/forgotPassword/styles.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../scss/combineStyles.scss';
+
+.modal {
+    font-family: $primary-font;
+    background-color: $bcg-gray-dark;
+    width: 380px;
+    border-radius: 6px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+
+    @media screen and (max-width: $width-sm) {
+        width: 100%;
+    }
+
+    .form {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        padding: 15px;
+        box-sizing: border-box;
+    }
+
+    .row {
+        display: flex;
+        flex-wrap: wrap;
+        margin: 15px 0 0 0;
+    }
+
+    .rowTop {
```

Comment:
We have another codestyle rules in css: row_top

Reply:
done